master/buildbot/newsfragments/gh-changehook-ci-skipping.feature

```diff
@@ -0,0 +1,9 @@
+Implement github change hook ci skipping (:issue:`3443`).
```

Comment:
the change log line must be small and point to a paragraph in the documentation. here ci is unknown word. try CI and maybe add it to ``master\docs\spelling_wordlist.txt``